src/main/java/com/treasure/hunt/view/CanvasController.java

```diff
@@ -67,6 +67,9 @@ public void onCanvasClicked(MouseEvent mouseEvent) {
         }
         offsetBackup = transformation.getOffset();
         dragStart = Vector2D.create(mouseEvent.getX(), mouseEvent.getY());
+        Vector2D mousePositionInGameContext = dragStart.subtract(offsetBackup);
+        mousePositionInGameContext = mousePositionInGameContext.multiply(1 / transformation.getScale());
+        System.out.println("recognized: " + gameManager.get().pickGeometryItem(mousePositionInGameContext.getX(), -mousePositionInGameContext.getY()).getGeometry());
```

Comment:
log never print on console

Reply:
Also how about showng this in UI, because without this it is kind of point less

Reply:
Let's use an event bus here

Reply:
https://www.baeldung.com/guava-eventbus

Reply:
This event can then be listened to by a widget or maybe the point inspector

Reply:
lets implement these in another feature.

Reply:
Ill handle this in a feature branch for #48